homeassistant/components/heos/__init__.py

```diff
@@ -75,12 +75,16 @@
         await controller.disconnect()
     hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, disconnect_controller)
 
+    # Get players and sources
     try:
-        players, favorites, inputs = await asyncio.gather(
-            controller.get_players(),
-            controller.get_favorites(),
-            controller.get_input_sources()
-        )
+        players = await controller.get_players()
```

Comment:
We could still gather this and `controller.get_input_sources`, right?

Reply:
We could, but it doesn't materially improve performance as there's still only 1 underlying telnet connection to the controller and these commands respond instantaneously.  I kept it separate to increase readability of the flow, but can gather it back up if desired.

Reply:
Ok, it's fine. I'll always comment on awaiting in loops, since sometimes it does make a difference.